bin/local/nuke-local.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+### failsafe logic to exit in case we are not running in our local minikube context
+KUBE_CONTEXT=$(kubectl config current-context)
+
+echo "Current kube context: '${KUBE_CONTEXT}'"
+
+if [[ "${KUBE_CONTEXT}" != "minikube-wbaas" ]]; then
+    echo "Error: wrong kube context. Use this script only within 'minikube-wbaas'!"
+    exit 1
+fi
+#####################################################################################
+
+# This script should re-create your local minikube cluster without requiring any additional input (can take about ~20 minutes)
+
+echo "⚠️ ☠️  NUKING your local cluster in 10 seconds ... all data will be lost! (press ctrl+c NOW to cancel) ☠️ ⚠️"
+sleep 10
+
+make minikube-delete
+make minikube-start
+
+cd tf/env/local/ && terraform apply -auto-approve # bad practice - you won't have a second chance to check for destructive actions
+
+cd ../../../k8s/helmfile/
```

Comment:
This means you can't call this script from any other working directory but `bin/local` if I'm not mistaken.

Maybe we can use sth like

```sh
cd $(dirname $0)/../../../k8s/helmfile/
```

instead so that it can be called from anywhere?

Reply:
the idea is to run these scripts from `wbaas-deploy/` root but this still makes sense to me, i'll test and add it